src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -0,0 +1,56 @@
+import { NextFunction, Request, Response } from "express";
+import { Errors } from "config/errors";
+import { Subscription } from "~/src/models/subscription";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+
+export const GithubCreateBranchOptionsGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+
+	const { jiraHost } = res.locals;
+	const { issue_key: key } = req.query;
+	if (!key) {
+		return next(new Error(Errors.MISSING_ISSUE_KEY));
+	}
+	const servers = await getGitHubServers(jiraHost);
```

Comment:
Probably worth catching jirahost up front before this call e.g.

`	if (!jiraHost) {
		return next(new Error(Errors.blah blah blah));
	}`